### PR TITLE
[Disease Browser]: The Disease Prevalence Rate Map

### PR DESCRIPTION
**Pull request by suhana13**
This PR fixes the disease prevalence rate map on the disease browser page. 
Screenshot 2023-08-30 at 12 32 38 PM

### CODE REVIEW

**Comment by chejennifer** [comment]
maybe update the styling to make the map section look better? Looks kind of odd to have everything so clustered on the left side of the page .. maybe move the legend to the right

File: static/css/biomedical/disease.scss [hunk 1]
```diff
@@ -60,3 +61,26 @@ tbody tr:nth-child(even) {
   stroke-width: 1.5;
   border: 3px solid #000;
 }
+
+.map {
```
Comment (chejennifer) [inline static/css/biomedical/disease.scss:65]: do you still need all these style changes after importing the tiles sheet?

**Review by chejennifer — APPROVED** [review]
nice! Before you merge, can you check if you even need the style additions you had made & if not, remove them

File: static/css/biomedical/disease.scss (hunk 1 above)
Comment (suhana13) [reply static/css/biomedical/disease.scss]: removed the unnecessary css elements!

**Pull request merged**